### PR TITLE
lt_lakes_req changes

### PR DESCRIPTION
**Pull request by KarHarker**
modified lt_lakes_sites to make the default option query active not all sites, moved 2 sites (E275784, 1132490) from inactive to active list 

#59

### CODE REVIEW

**Review by ateucher — COMMENTED** [review]
Looks great! I think we can keep working through the checklist and merge it all as one once done.

File: R/helpers.R [hunk 1]
```diff
@@ -47,22 +47,42 @@ lt_lake_sites <- function(which = c("all", "active", "inactive")) {
...
 }
+
+
+#' Long-term lake monitoring sites
+#'
+#' Get the REQ_IDs of all of the long-term lake monitoring sites.
+#'
+#' @param data loads a site ID lookup table
+#'
+#' @return a character vector of req ids (EMS IDs)
```
Comment (ateucher) [inline R/helpers.R]: It says EMS IDs here
```suggestion
#' @return a character vector of requisition ids 
```

File: R/data_ops.R [hunk 2]
```diff
@@ -32,11 +32,12 @@ bind_ems_data <- function(...) {
 #' @param param_code a character vector of parameter codes
 #' @param from_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
 #' @param to_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
+#' @param req_id A character vector of requisition id(s) of interest
```
Comment (ateucher) [inline R/data_ops.R]: Could add a reference to `lt_lake_req()` - If you put it in square brackets roxygen will create a hyperlink to the man page for that function (e.g., "`See [lt_lakes_req()] for a list of long-term lakes monitoring sites requisition IDs`")

**Comment by KarHarker** [comment]
ok @ateucher - I think this is complete!

File: R/data_ops.R [hunk 3]
```diff
@@ -32,11 +32,12 @@ bind_ems_data <- function(...) {
 #' @param param_code a character vector of parameter codes
 #' @param from_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
 #' @param to_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
+#' @param req_id A character vector of requisition id(s) of interest. See [lt_lake_req()] for a list on long-term lakes monitoring sites REQUISITION IDs
```
Comment (ateucher) [inline R/data_ops.R]:
```suggestion
#' @param req_id A character vector of requisition id(s) of interest. See [lt_lake_req()] for a list of long-term lakes monitoring sites REQUISITION IDs
```

**Comment by ateucher** [comment]
Looks great @KarHarker! I fixed one small typo - so if you can pull and run `document()` and push it back up, once the checks all pass go ahead and merge!

**Pull request merged**